core/storage/base_model/gae_models.py

```diff
@@ -537,16 +544,17 @@ def get_snapshots_metadata(
         error is raised.
 
         Args:
-            model_instance_id: Id of requested model.
-            version_numbers: List of version numbers.
+            model_instance_id: str. Id of requested model.
+            version_numbers: list(int). List of version numbers.
             allow_deleted: bool. If is False, an error is raised if the current
                 model has been deleted. Default is False.
 
         Returns:
-            List of dicts, each dict representing a model snapshot.
+            list(dics). Each dict representing a model snapshot.
 
         Raises:
-            Exception: If any of version numbers does not exist.
+            Exception: There is no model instance corresponding to at least one
+                of the given version numbers.
         """
         # pylint: disable=protected-access
```

Comment:
NB There are a few methods below (see BaseSnapshotMetadataModel etc.) that don't seem to be documented?